tests/virt/cluster/common_templates/custom_namespace/test_custom_namespace.py

```diff
@@ -16,7 +16,7 @@
 
 LOGGER = logging.getLogger(__name__)
 
-pytestmark = pytest.mark.post_upgrade
+pytestmark = [pytest.mark.post_upgrade, pytest.mark.gating]
```

Comment:
Some tests in this file are not good gating candidate. Please re-review and remove those. 1) negative tests, 2) tests that won't block release are not good candidate for gating.

Reply:
+1